### PR TITLE
Python transition: clean up stale keywords

### PR DESCRIPTION
**Pull request by krnowak**
~This pulls in #1269 and #1261, so this PR is blocked on those.~

Most of the package updates happened in portage-stable. Here:

- we mostly drop stale entries from the accept_keywords file.
- tell `app-emulation/google-compute-engine` to use `python3` explicitly.
- ~move `app-text/asciidoc` here from portage-stable, so we can apply an extra patch from gentoo.~
  - ~the package can be move back later, when we have a newer version of python3, so we could also update `app-text/asciidoc`~

Comes with https://github.com/flatcar-linux/portage-stable/pull/214 and https://github.com/flatcar-linux/scripts/pull/152.

CI: http://localhost:9091/job/os/job/manifest/4229/cldsv/

### CODE REVIEW

File: app-text/asciidoc/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+The sole reason for keeping this package in overlay is to apply a
+patch from upstream gentoo that fixes the tarball location to
+something more predictable. The gentoo commit is
+66358a5d95c84fcf8958098b5e71076d4622ecfc.
```
[the remark this answers is not in the document]
Comment (krnowak) [reply app-text/asciidoc/README.md]: asciidoc is back in portage-stable with modified PYTHON_COMPAT stuff.

File: sys-apps/systemd/systemd-9999.ebuild [hunk 2]
```diff
@@ -498,18 +502,18 @@ multilib_src_install_all() {
...
-    )
+builddir_systemd_enable_service() {
+	local target=${1}
+	local service=${2}
+	local ud=$(builddir_systemd_get_systemunitdir)
+	local destname=${service##*/}
+
+	dodir "${ud}"/"${target}".wants && \
+	dosym ../"${service}" "${ud}"/"${target}".wants/"${destname}"
 }
```
[the remark this answers is not in the document]
Comment (krnowak) [reply sys-apps/systemd/systemd-9999.ebuild]: This PR doesn't touch systemd (it was a part of pkg-config impl update, which is already merged), so it's not a problem any more.

**Comment by krnowak** [comment]
CI passed.

**Comment by krnowak** [comment]
Please hold on with reviews on this one - after talking with @jepio I decided to also put a python version update into it. So here I'll add an update of python-oem too.

**Review by jepio — APPROVED** [review]
LGTM. As long as CI passes and merge conflicts are resolved I'm fine with merging. It's a lot of commits but I actually looked through all of them and most are very small and mechanic in nature.

**Comment by krnowak** [comment]
Thanks for the review. CI partially succeeded - os/kola/qemu passed, but os/kola/azure did not and I need to find out why (not much to see in logs, though - http://localhost:9091/job/os/job/kola/job/azure/1098/console).

**Pull request merged**